### DISCUSSION TITLE
Fabric not loading right

### DISCUSSION THREAD

**Discussion by chessbored** [FabricMC/fabric-api#2173] [Player support] [answered]
ive been trying to play with my friends on modded Fabric servers, but anytime i try it wont load. I have restarted my computer, redownloaded the mods, Fabric AND Minecraft, my drivers are all up to date and so is my computer. whats wrong with it? this is the output it gives me for one of the modpacks. pastebin.com/apht2f07
and yes, ive tried putting even almost all my RAM into it. it doesnt wanna work either way.

**Comment by warjort** [comment]
````
Invalid floating point value for option textBackgroundOpacity = 
java.lang.NumberFormatException: empty String
````
See: https://github.com/FabricMC/fabric/issues/2126

**Comment by warjort** [comment]
> Process crashed with exit code -805306369

This is your real crash which is an out of memory error per this:
https://minecrafthopper.net/help/exit-code/code-805306369/

**Comment by warjort** [comment, marked as answer]
While this gives other things to try:
https://bugs.mojang.com/browse/MCL-6550

**Reply by chessbored** [reply]
thanks, this helped :D the issue was Razer Synapse
